controllers/operands/kubevirt.go

```diff
@@ -521,22 +521,18 @@ func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfigu
 	return config, nil
 }
 
-func getNetworkBindings(
-	hcoNetworkBindings map[string]kubevirtcorev1.InterfaceBindingPlugin,
-	hcoFeatureGates hcov1beta1.HyperConvergedFeatureGates) map[string]kubevirtcorev1.InterfaceBindingPlugin {
+func getNetworkBindings(hcoNetworkBindings map[string]kubevirtcorev1.InterfaceBindingPlugin) map[string]kubevirtcorev1.InterfaceBindingPlugin {
 	networkBindings := maps.Clone(hcoNetworkBindings)
 
-	if hcoFeatureGates.PrimaryUserDefinedNetworkBinding != nil && *hcoFeatureGates.PrimaryUserDefinedNetworkBinding {
-		if networkBindings == nil {
-			networkBindings = make(map[string]kubevirtcorev1.InterfaceBindingPlugin)
-		}
-
-		sidecarImage, _ := os.LookupEnv(hcoutil.PrimaryUDNImageEnvV)
-		networkBindings[primaryUDNNetworkBindingName] = primaryUserDefinedNetworkBinding()
-		networkBindings[passtUDNNetworkBindingName] = passtUserDefinedNetworkBinding(sidecarImage)
+	if networkBindings == nil {
+		networkBindings = make(map[string]kubevirtcorev1.InterfaceBindingPlugin)
 	}
-	return networkBindings
 
+	sidecarImage, _ := os.LookupEnv(hcoutil.PrimaryUDNImageEnvV)
+	networkBindings[primaryUDNNetworkBindingName] = primaryUserDefinedNetworkBinding()
+	networkBindings[passtUDNNetworkBindingName] = passtUserDefinedNetworkBinding(sidecarImage)
```

Comment:
do we actually want to do this ? AFAICT we don't want to ship passt in 4.18 

@EdDev / @phoracek

Reply:
was concluded that we keep passt for not few times fwiw

Reply:
> imho even if we want to remove it, we should not be done on this PR please
best to keep the PR focused on one thing

OK.